nav2_gps_waypoint_follower_demo/config/nav2_no_map_params.yaml

```diff
@@ -247,19 +249,48 @@ global_costmap:
 #   ros__parameters:
 #     yaml_filename: ""
 
+keepout_filter_mask_server:
```

Comment:
Can these be removed since they're not used in this demo? This and the new things below

Reply:
done in  https://github.com/ros-navigation/navigation2_tutorials/pull/115/commits/fd521ea5a04418d8a265ed30ecb6cd07d40dcc01